tests/cases/fourslash/completionsImport_named_didNotExistBefore.ts

```diff
@@ -31,9 +31,7 @@ verify.completions({
         },
     ], { noLib: true }),
     preferences: { includeCompletionsForModuleExports: true },
-});
-
-verify.applyCodeActionFromCompletion("", {
+}).andApplyCodeAction({
```

Comment:
This is a new fourslash API for calling `completionEntryDetails` with the exact `data` that was returned in the `completionInfo` response, as an editor would do. I was considering making sending `data` for auto-import completion details requests mandatory, breaking super old editors and about 40 old tests like this one, but ultimately decided against it.